### ISSUE TITLE
Cant get the virtual grid to start, I am using a build for Windows made by rackedge.

### ISSUE THREAD

**Issue by Zugelno** [Dewb/monome-rack#29]
### What's the behavior you observed? I tried the White Wale module and virtual grid 64
right clicked on White Wale and connected it to virtual grid 64.
And a clock in on  White Wale.
But when i click on the virtual grid, nothing happens.
What are I am doing wrong.
Do you have a vcv rack patch I could download to see the connections

### If you're having trouble building, please paste the output of these commands:
   ```
   git status
   git submodule status
   make clean
   make
   ```

**Comment by Dewb** [comment]
I'm curious to take a look, where is your builds repo?

**Comment by Zugelno** [comment]
Hello, thanks for fast reply. I am not a coder, so how do I get the terminal log output from the Rack session?
Do you have a patch from the Rack witch contains the connected modules?
Many thanks for making these modules.

**Comment by phdsg** [comment]
@Dewb it's here: https://github.com/phdsg/rack-edge
@Zugelno if you're not starting rack from a console you should still find a log.txt in the documents folder rack uses. on windows that's usually C:\Users\yourname\Documents\Rack.

**Comment by Zugelno** [comment]
Here is the log
[log.txt](https://github.com/Dewb/monome-rack/files/1628354/log.txt)

**Comment by Zugelno** [comment]
And here is the log from newly installed Rack 0.5.1 not working either.
[log.txt](https://github.com/Dewb/monome-rack/files/1628376/log.txt)

**Comment by phdsg** [comment]
the firmware is expecting (hardcoded path) that this plugin's directory is called 'monome-rack' (which it is on my side, so it works for me), but the dist target in the makefile creates a directory called 'Monome' which leads to the obvious conflict. i will fix the makefile for my builds so it creates the correct directory.

**Comment by Zugelno** [comment]
Ok, many thanks. Looking forward to try it. Have a nice weekend.

**Comment by Zugelno** [comment]
That worked ! It seems like that this build is working. The 3 modules lights up and the virtual grids also lights up and advancing steps. Many many thanks for the help. Now its time to learn monome:). Have a nice weekend.
Best from Norway. Many thanks to you dewb for building these modules.

**Comment by Dewb** [comment]
@phdsg Yup, I haven't made dist builds work yet. It's next on the list after #7.

@Zugelno Glad it works for you! Sorry for the runaround, this will get a lot easier once the package is released. Thanks for beta testing, please open another issue if you spot any new problems.

**Comment by Zugelno** [comment]
Ok, thats ok, now its time to learn Monome, steep learning curve. Ill let you now if I spot any problems. Thanks. Keep up the god work, VCV Rack is so fun, I cant believe its free.
